### ISSUE TITLE
`make quicklint` no longer runs `mypy`

### ISSUE THREAD

**Issue by awgu** [pytorch/pytorch#76063]
I am not sure if this is the right place to ask, but I think after https://github.com/pytorch/pytorch/pull/68460, `make quicklint` no longer runs `mypy`. Is this change intentional? What is the recommended way to run `mypy` only on locally-changed files?

cc @ezyang @malfet @rgommers @xuzhao9 @gramster

**Comment by mruberry** [comment]
cc @suo

**Comment by ezyang** [comment]
you can do `make lint` to get mypy; idk what quicklint is supposed to be for

**Comment by suo** [comment]
make lint and make quicklint will both run mypy, through lintrunner. In both cases, it will only run on your changed files.

check out https://github.com/pytorch/pytorch/wiki/lintrunner to get set up with it

**Comment by awgu** [comment]
Thanks for the responses everyone! I am not sure exactly what happened earlier when I filed this issue, but I can now run `mypy` with `lintrunner`.

One minor thing was that I had to change the `numpy==1.20` requirement in `.lintrunner.toml` to `numpy==1.19` since `lintrunner init` was erroring saying `ERROR: Could not find a version that satisfies the requirement numpy==1.20`. Mentioning this here in case someone else encounters the same error.
